internal/store/models/storedobject.go

```diff
@@ -17,44 +17,38 @@ package models
 
 // StoredObject is the atomic and most abstract description of what is collected by the export store system.
 type StoredObject struct {
-	// id is the unique identifier for this record once stored.
-	id string
+	ID string `bson:"id"`
```

Comment:
Not wild about putting the Mongo specifics in the generic model. Is there a better way to handle this in the mongo implementation?

Reply:
This is how Go does unmarshalling; with tags on the model. Redis will presumably add its own tags, as will JSON unmarshalling and whatever else.

Reply:
Not seeing this on the models in edgex-go. Lets discuss in today's WG call.

Reply:
@lenny-intel Here's an example from the Mongo models in edgex-go
https://github.com/edgexfoundry/edgex-go/blob/f07acc231909531283c898e4100e3cc5d499f8b9/internal/pkg/db/mongo/models/device.go#L32

Reply:
@lenny-intel I think you're looking at the contract and not the model, which we're trying to avoid having a contract and a model.

Reply:
@tsconn23 , OK so there is a separate `mongo` model from the abstraction model. Thx.